### DISCUSSION TITLE
APOE: regression run snagged

### DISCUSSION THREAD

**Discussion by krootesmurdy** [trendscenter/coinstac#1527] [General]
Tried running two sites but the regression just kept running, never moved past iteration 1.

Solution: Javier reset the server (no change to the files or the pipeline).

**Comment by rssk** [comment]
can we get UI logs for this?

**Reply by rssk** [reply]
for the setup: ```the setup was done two different ways. I mapped two sites from two machines and Javier mapped one site from one machine, it hung for over an hour. I kicked him out and cleared data, and re-mapped with my two sites from two machines, still hung.```
